roofit/roofitmore/inc/LinkDef1.h

```diff
@@ -0,0 +1,13 @@
+#ifdef __CINT__ 
+#pragma link off all globals;
+#pragma link off all classes;
+#pragma link off all functions;
+
+#pragma link C++ class RooAdaptiveGaussKronrodIntegrator1D+; 
```

Comment:
This line should be removed from `roofit/roofitcore/inc/LinkDef1.h` else we have two entries for that class.